### ISSUE TITLE
Bei der Wertpapierumbuchung Kurs nicht anzeigen

### ISSUE THREAD

**Issue by markokocic** [buchen/portfolio#1391]
Bei der Umbuchung der Wertpapiere vom Depot nach Depot sollte kein Kurs gezeigt werden.

Kurs am Tag der Umbuchung hat keine Relevanz für die Gewinn- und Kurserfolgsrechnung.

Nur Einkaufskurs und Datum sind relevant als in #1292 beschrieben.

**Comment by Ragas13** [comment]
@markokocic 
Das ist so leider nicht ganz richtig, wenn nur einzelne Depots gefiltert betrachtet wird kann bei Umbuchung der Anfangswert nicht errechnet werden.

**Comment by markokocic** [comment]
> Das ist so leider nicht ganz richtig, wenn nur einzelne Depots gefiltert betrachtet wird kann bei Umbuchung der Anfangswert nicht errechnet werden.

Wenn bei der Umbuchung Einstandkurs statt Umbuchungsfachkurses stünde, könnten die Auswertungen der Kurserfolge richtig gezeigt werden. Manche Broker machen einfach so.

> P.S. Beim Übertragenen des Depots übernimmt die neue Bank ja auch nicht nur Stückzahlen 😉

Depotübertrag ist kein steuerrelevantes Ereignis und hat kein Einfluss auf die Gewinn- oder Steuerberechnung des Wertpapiers.

Beim Übertagungen des Depots übernimmt der neue Broker neben der Stückzahl auch den Einkaufskurs und Einkaufsdatum (wegen FIFO Berechnung). Für die Auswertungen und Steuerberechnungen werden dann Einkaufskurs und Datum genützt, nicht Umbuchungskurs und -Datum.

Wenn Umbuchungskurs und Datum relevant wären, könnte man den Fiskus umgehen, indem man einen Depotübertrag kurz vor Verkauf macht, und danach alle ohne Steuer verkaufen, weil der Umbuchungskurs quasi das gleiche wie Verkaufskurs wäre.

**Comment by Ragas13** [comment]
Dann sind wir uns mE zumindest darüber einig, dass hier die FIFO Kurse des angehenden Depots besser wären?

**Comment by markokocic** [comment]
Ja, die ganzen FIFO Daten werden beim Umbuchung übertragen; diese können gezeigt aber nicht geändert werden, weil sie von den Kaufwerten stammen.

Aber um das alles sauber zu lösen, sollte man auch die Gewinnberechnung in #1292 anpassen, damit die FIFO und nicht Umbuchung Werten benutzt werden.

**Comment by Ragas13** [comment]
**Für die Akte**
Ich meine es war mal den Transfers so, dass dich der Kurs auf den Tag des Transfers bezogen hatte,  dies wurde aber mit https://github.com/buchen/portfolio/commit/6173576dc8953079d53025c4bce2270db9f5fc25 deaktiviert.

Und ja, im Forum @ https://forum.portfolio-performance.info/t/buchungsbetraege-werden-beim-editieren-von-buchungsdepot-oder-konto-veraendert/1213 gab es das Problem, wenn Buchungen bearbeitet wurden hat sich der Kurs geändert.

@buchen 
Wäre es beim Transfer only evtl möglich, auf den FIFO Kurs vom Tag der Buchung vom abgebenden Portfolio zu verwenden? Es darf natürlich nicht vom Gesamtwert zurück gerechnet werden sondern mE nur Stückzahl × FIFO Abgebendes Portfolio.

**Comment by Flippo24** [comment]
Schaut mal [hier ](https://www.clearstream.com/resource/blob/1314920/1e6a07363ceade35c6994c1d2c3613d2/ti03-frage-antwort-katalog-de-data.pdf)rein. Unter Punkt 1.3 ist geschrieben, dass bei einem Gläubiger-Wechsel eine fiktive Veräußerung statfindet und die Verußerungswerte die neuen Anschaffungsdaten sind.

**Comment by markokocic** [comment]
Gläubiger-Wechsel ist Ausnahmefall und gilt nur wenn zwei Depots verschiedene Eigentümer haben. 

Normalfall ist die Übertragung auf eigenes Depot ohne Besitzer-Wechsel. 

Punkt 1.3 sagt dazu: "Bei Gläubiger-Identität liegt Personengleichheit vor 
und es sind die ursprünglichen Anschaffungsdaten 
zu übermitteln."

**Comment by Flippo24** [comment]
Jetzt weiß aber PP nicht ob das Die Depots verschiedenen Personen gehören. Z.B. Mann, Frau, Kind...

**Comment by buchen** [comment]
> Bei der Umbuchung der Wertpapiere vom Depot nach Depot sollte kein Kurs gezeigt werden.

PP braucht für den Zeitpunkt der Umbuchung auf jeden Fall eine Bewertung der Position. @Ragas13 hat ja schon einen Fall beschrieben: wenn man nach Depots filtert, dann ist das plötzlich einen neue Position die PP bewerten können muss.

Natürlich könnte PP sagen: ich nehme einfach den Tageschlusskurs. Aber der kann ja nicht vorhanden sein, könnte von Benutzer geändert werden, etc. - darum frage ich einfach explizit.

> Wenn bei der Umbuchung Einstandkurs statt Umbuchungsfachkurses stünde, könnten die Auswertungen der Kurserfolge richtig gezeigt werden. Manche Broker machen einfach so.

In dem Eingabedialog zu einer Wertpapierbuchung steht ja nur "Kurs" - was ja auch für Umbuchungsfachkurs stehen könnte.

Wo sollte den Einstandskurs nicht angezeigt werden? Vielleicht könnte man den Begriff ändern?

**Comment by buchen** [comment]
Unabhängig bin ich noch nicht zu #1292 gekommen. Leider.

**Comment by markokocic** [comment]
> > Bei der Umbuchung der Wertpapiere vom Depot nach Depot sollte kein Kurs gezeigt werden.
> 
> PP braucht für den Zeitpunkt der Umbuchung auf jeden Fall eine Bewertung der Position. @Ragas13 hat ja schon einen Fall beschrieben: wenn man nach Depots filtert, dann ist das plötzlich einen neue Position die PP bewerten können muss.

Wenn man das angehende Depot bewertet, können die FIFO Daten übertragener Wertpapiere bewertet, als ob die immer dagewesen sind. Zumindest machen Comdirect und Consors bei ihrer Depotbewertung so.

> In dem Eingabedialog zu einer Wertpapierbuchung steht ja nur "Kurs" - was ja auch für Umbuchungsfachkurs stehen könnte.

> Wo sollte den Einstandskurs nicht angezeigt werden? Vielleicht könnte man den Begriff ändern?

Bei der Umbuchung es geht nicht nur um Begriff. Idealerweise würden bei dem Übmuchungsdialog mehrere Kurse und Stückelungen von den nach FIFO übertragenen Stückzahl gezeigt, ähnlich wie in der "Trade" Liste, denn das sind die übertragenen Daten.

Aber, natürlich, dafür muss man auch #1292 betrachten.
